XiEditor/StatusBar.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  StatusBar.swift
+//  XiEditor
+//
+//  Created by Dzũng Lê on 19/05/2018.
+//  Copyright © 2018 Raph Levien. All rights reserved.
+//
+
+import Foundation
+import Cocoa
+
+enum StatusItemAlignment: String {
+    case left = "left"
+    case right = "right"
+}
+
+class StatusItem: NSTextField {
+
+    var key: String = ""
```

Comment:
perhaps these should be `let` bindings? In which case you don't need to assign them initial values, if  you assign them values in `init`.